specs/HubProtocol.md

```diff
@@ -97,7 +98,18 @@ If either endpoint commits a Protocol Error (see examples below), the other endp
 * It is a protocol error for a Caller to send a `Completion` message carrying both a result and an error.
 * It is a protocol error for an `Invocation` or `StreamInvocation` message to have an `Invocation ID` that has already been used by *that* endpoint. However, it is **not an error** for one endpoint to use an `Invocation ID` that was previously used by the other endpoint (allowing each endpoint to track it's own IDs).
 
-## Examples
+## Ping (aka "Keep Alive")
+
+The SignalR Hub protocol supports "Keep Alive" messages used to ensure that the underlying transport connection remains active. These messages help ensure:
+
+1. Proxies don't close the underlying connection during idle times (when few messages are being sent)
+1. If the underlying connection is dropped without being terminated gracefully, the application is informed as quickly as possible.
```

Comment:
nit: 2.

Reply:
FYI, Markdown doesn't actually require that, it regenerates the numbers. I generally prefer to use `1.` so that it's not order-sensitive. Regardless, it's a trivial change and I'm happy to make it.